lib/TabTracker.js

```diff
@@ -76,7 +79,7 @@ TabTracker.prototype = {
 
   logActivate: function(tab) {
     if (this.isActivityStreamsURL(tab.url)) {
-      ss.storage.tabData[tab.url][tab.id].activations.push({startTime: Date.now()});
+      this._tabData[tab.url][tab.id].activations.push({startTime: Date.now()});
```

Comment:
I'm going to change this entire structure to align with what we talked about today.This pull request was just for switching to using memory rather than simple storage and for notifications.